versions.yaml

```diff
@@ -18,6 +18,9 @@ module-sets:
     modules:
       - go.opentelemetry.io/collector
       - go.opentelemetry.io/collector/cmd/builder
+      - go.opentelemetry.io/collector/exporter/loggingexporter
```

Comment:
Was this added manually? Or is this the one that is managed by multimod?

Reply:
Manually to get `multimod-verify` to pass